packages/app/src/runs/modals/SelectCloudProjectModal.vue

```diff
@@ -294,7 +294,13 @@ const pickedOrganization = ref(organizations.value.length >= 1 ? organizations.v
 
 const projects = computed(() => pickedOrganization.value?.projects?.nodes || [])
 const newProject = ref(projects.value.length === 0)
-const pickedProject = ref(pickedOrganization.value?.projects ? pickedOrganization.value.projects.nodes.find((p) => p.name === projectName.value) : undefined)
+const pickedProject = computed(() => {
```

Comment:
I think switching this from a `ref` to `computed` broke the `v-model` aspects when an organization has multiple projects. I'm not able to pick a different project from the menu